composer.json

```diff
@@ -9,7 +9,7 @@
     "homepage": "https://laminas.dev",
     "license": "BSD-3-Clause",
     "require": {
-        "php": "^7.4 || ^8.0",
+        "php": "^7.4 || ^8.0 || ^8.1 || ^8.2",
```

Comment:
`^8.0` already includes 8.1 and 8.2 :)

Reply:
Fixed. Could you review https://github.com/laminas-api-tools/api-tools-configuration/pull/19 ?